### PR TITLE
K8SPG-597 Update images and versions for 2.4.0 release

### PR DESCRIPTION
**Pull request by ptankov**
[K8SPG-597](https://jira.percona.com/browse/K8SPG-597) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

**CHANGE DESCRIPTION**
---
**Problem:**
*Short explanation of the problem.*

**Cause:**
*Short explanation of the root cause of the issue if applicable.*

**Solution:**
*Short explanation of the solution we are providing with this PR.*

**CHECKLIST**
---
**Jira**
- [ ] Is the Jira ticket created and referenced properly?
- [ ] Does the Jira ticket have the proper statuses for documentation (`Needs Doc`) and QA (`Needs QA`)?
- [ ] Does the Jira ticket link to the proper milestone (Fix Version field)?

**Tests**
- [ ] Is an E2E test/test case added for the new feature/change?
- [ ] Are unit tests added where appropriate?

**Config/Logging/Testability**
- [ ] Are all needed new/changed options added to default YAML files?
- [ ] Did we add proper logging messages for operator actions?
- [ ] Did we ensure compatibility with the previous version or cluster upgrade process?
- [ ] Does the change support oldest and newest supported PG version?
- [ ] Does the change support oldest and newest supported Kubernetes version?

[K8SPG-597]: https://perconadev.atlassian.net/browse/K8SPG-597?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: deploy/cr.yaml [hunk 1]
```diff
@@ -105,7 +105,7 @@ spec:
 #          test-label: value
 
 
-  image: perconalab/percona-postgresql-operator:main-ppg16-postgres
+  image: percona/percona-postgresql-operator:2.4.0-ppg16-postgres
```
Comment (hors) [inline deploy/cr.yaml:108]: hi, @ptankov, please add a minor version of PG to tag as well. As I remember you even have a task for it.

**Review by hors — CHANGES_REQUESTED** [review]

File: deploy/cr.yaml (hunk 1 above)
Comment (ptankov) [reply deploy/cr.yaml]: done

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| custom-extensions | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/7c6cae060/custom-extensions.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/7c6cae060/demand-backup.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/7c6cae060/init-deploy.log) |
| major-upgrade | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/7c6cae060/major-upgrade.log) |
| monitoring | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/7c6cae060/monitoring.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/7c6cae060/one-pod.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/7c6cae060/operator-self-healing.log) |
| pitr | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/7c6cae060/pitr.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/7c6cae060/scaling.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/7c6cae060/scheduled-backup.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/7c6cae060/self-healing.log) |
| start-from-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/7c6cae060/start-from-backup.log) |
| tablespaces | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/7c6cae060/tablespaces.log) |
| telemetry-transfer | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/7c6cae060/telemetry-transfer.log) |
| upgrade-consistency | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/7c6cae060/upgrade-consistency.log) |
| upgrade-minor | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/7c6cae060/upgrade-minor.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/7c6cae060/users.log) |
| We run 17 out of 17|

commit: https://github.com/percona/percona-postgresql-operator/pull/819/commits/7c6cae06049e9dec877c9f8826561958b46416bd
image: `perconalab/percona-postgresql-operator:PR-819-7c6cae060`

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| custom-extensions | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/7c6cae060/custom-extensions.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/7c6cae060/demand-backup.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/7c6cae060/init-deploy.log) |
| major-upgrade | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/7c6cae060/major-upgrade.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/7c6cae060/monitoring.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/7c6cae060/one-pod.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/7c6cae060/operator-self-healing.log) |
| pitr | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/7c6cae060/pitr.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/7c6cae060/scaling.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/7c6cae060/scheduled-backup.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/7c6cae060/self-healing.log) |
| start-from-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/7c6cae060/start-from-backup.log) |
| tablespaces | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/7c6cae060/tablespaces.log) |
| telemetry-transfer | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/7c6cae060/telemetry-transfer.log) |
| upgrade-consistency | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/7c6cae060/upgrade-consistency.log) |
| upgrade-minor | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/7c6cae060/upgrade-minor.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/7c6cae060/users.log) |
| We run 17 out of 17|

commit: https://github.com/percona/percona-postgresql-operator/pull/819/commits/7c6cae06049e9dec877c9f8826561958b46416bd
image: `perconalab/percona-postgresql-operator:PR-819-7c6cae060`

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| custom-extensions | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/aeba89dc2/custom-extensions.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/aeba89dc2/demand-backup.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/aeba89dc2/init-deploy.log) |
| major-upgrade | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/aeba89dc2/major-upgrade.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/aeba89dc2/monitoring.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/aeba89dc2/one-pod.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/aeba89dc2/operator-self-healing.log) |
| pitr | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/aeba89dc2/pitr.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/aeba89dc2/scaling.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/aeba89dc2/scheduled-backup.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/aeba89dc2/self-healing.log) |
| start-from-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/aeba89dc2/start-from-backup.log) |
| tablespaces | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/aeba89dc2/tablespaces.log) |
| telemetry-transfer | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/aeba89dc2/telemetry-transfer.log) |
| upgrade-consistency | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/aeba89dc2/upgrade-consistency.log) |
| upgrade-minor | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/aeba89dc2/upgrade-minor.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-819/aeba89dc2/users.log) |
| We run 17 out of 17|

commit: https://github.com/percona/percona-postgresql-operator/pull/819/commits/aeba89dc2b9e8f09207bc9a25331f0ccd11c2725
image: `perconalab/percona-postgresql-operator:PR-819-aeba89dc2`

**Review by hors — APPROVED** [review]

**Pull request merged**